ui/src/index.css

```
@@ -2982,36 +2982,324 @@ table.auto-table {
+}
+
+.game-fs-tip {
+  font-size: 0.85rem;
+  color: #64748b;
 }
 
-.game-card-fs {
-  width: 340px;
-  cursor: default;
-  transform-origin: center center;
+@media (max-width: 768px) {
```

Stylelint (17.14.0)

[error] Expected "context" media feature range notation (media-feature-range-notation)

(media-feature-range-notation)